mobify/platform-scaffold
# Web 1426  add from wishlist

This PR adds the ability to add an item to your cart from your wishlist. 


 **JIRA**: https://mobify.atlassian.net/browse/WEB-1426
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/779

## Changes
- Added UI for add to cart success
- Added Ui for add to cart failure
- refactored contents of product details item added modal into a re-usable component
- Added merlins command for addToCartFromWishlist
- Added SFCC commend to addToCartFromWishlist & removeItemFromWishlist 


## To dos
- [x] Add command to stub connector

## How to test-drive this PR
- checkout the add-from-wishlist-command branch of the SDK
- run `npm run dev:build`
- run `npm link`
- cd into the scaffold web folder
- run `npm link progressive-web-sdk`
- preview the merlin's site
- sign in to an account with items in the wishlist
- view the wishlist page
- add an item from the wishlist to the cart
- the item should be added to the cart and removed from the wishlist
- repeat using the SFCC connector & previewing the SFCC site
